src/genie/libs/parser/iosxe/tests/ShowIpSlaSummary/cli/equal/golden_output_1_expected.py

```diff
@@ -0,0 +1,39 @@
+expected_output = {
+    'probes': {
+            '1': {'destination': '123.23.213.32',
+                  'last_run': '21 seconds ago',
```

Comment:
I would suggest to have kind of unit in key name and have just number as integer.
```suggestion
                  'last_run_seconds_ago': 21,
```

Reply:
confirmed the change